default.build

```diff
@@ -46,32 +46,32 @@
 		</exec> 
 	</target>
 
-	<target name="solution-restore" depends="common.nuget-restore">
+	<target name="solution-restore" depends="common.tools-restore">
 		<exec program="dotnet" verbose="true">
 			<arg value="msbuild" />
 			<arg value="/t:Restore" />
 			<arg value="${root.dir}/src/NHibernate.sln" />
 		</exec> 
 	</target>
 
-	<target name="get-nuget-package-path" depends="common.nuget-restore">
+	<target name="get-tool-info" depends="common.tools-restore">
 		<xmlpeek 
-			file="${tools.dir}/packages.config" 
-			xpath="/packages/package[@id = '${nuget-package-id}']/@version"
-			property="nuget-package-version" />
-		<property name="nuget-package-path" value="${tools.dir}/${nuget-package-id}.${nuget-package-version}/" />
+			file="${tools.dir}/packages.csproj" 
+			xpath="/Project/ItemGroup/PackageReference[@Include = '${tool.id}']/@Version"
+			property="tool.version" />
+		<property name="tool.path" value="${tools.dir}/${tool.id}/${tool.version}/" />
 	</target>
 
 	<target name="find-async-generator-console">
-		<property name="nuget-package-id" value="CSharpAsyncGenerator.CommandLine" />
-		<call target="get-nuget-package-path" />
-		<property name="async-generator-console"  value="${nuget-package-path}/tools/AsyncGenerator.CommandLine.exe" />
+		<property name="tool.id" value="CSharpAsyncGenerator.CommandLine" />
+		<call target="get-tool-info" />
+		<property name="async-generator-console"  value="${tool.path}/tools/netcoreapp2.1/AsyncGenerator.CommandLine.dll" />
```

Comment:
Small glitch here: `tool.path` always includes a final `/`, so adding one right after it causes a double `//`.

Reply:
Fixed.